.gitignore

```diff
@@ -1,6 +1,7 @@
 .DS_Store
 .idea
 .vscode
+*.test
```

Comment:
gru?

Reply:
When running tests I sometimes get these `<module>.test` files (like `vparquet2.test`) and I honestly have no idea what they do, but they are never committed and cause noise in the git status. Googling for it was unhelpful.